### PR TITLE
Issue 1079  update tgv

### PR DESCRIPTION
**Pull request by adler-j**

### CODE REVIEW

File: examples/solvers/chambolle_pock_tomography_tgv.py [hunk 1]
```diff
@@ -47,12 +47,7 @@
 # --- Generate artificial data --- #
 
 # Create phantom
-def phantom_func(pts):
-    x, y = pts
-    sign = ((x < -10) | (x > 10) | (y < -10) | (y > 10)) * 3 - 2
-    return 0.5 * ((x / 20.0) * sign + 1)
-
-phantom = U.element(phantom_func)
+phantom = odl.phantom.tgv_phantom(U)
```
Comment (kohr-h) [inline examples/solvers/chambolle_pock_tomography_tgv.py]: Can we go with the usual `reco_space` here? Calling `A` something else like `fwd_op` would also be nice.

File: odl/phantom/geometric.py [hunk 2]
```diff
@@ -687,6 +687,9 @@ def tgv_phantom(space):
     space : `DiscreteLp`, 2 dimensional
         Discretized space in which the phantom is supposed to be created.
         Needs to be two-dimensional.
+    edge_smoothness : float, optional
+        Smoothing of the edges of the phantom.
+        Given in the width of the smoothing in units of pixels.
```
Comment (kohr-h) [inline odl/phantom/geometric.py]: Looking at the implementation, perhaps "[...] phantom, given as smoothing width in units of minimum pixel size."

File: odl/phantom/geometric.py [hunk 3]
```diff
@@ -687,6 +687,9 @@ def tgv_phantom(space):
     space : `DiscreteLp`, 2 dimensional
         Discretized space in which the phantom is supposed to be created.
         Needs to be two-dimensional.
+    edge_smoothness : float, optional
```
Comment (kohr-h) [inline odl/phantom/geometric.py]: nonnegative

File: examples/solvers/chambolle_pock_tomography_tgv.py (hunk 1 above)
Comment (adler-j) [inline examples/solvers/chambolle_pock_tomography_tgv.py]: @MatthiasJE used this notation so i followed him. I say we leave this for now.

**Pull request merged**